_includes/head.html

```diff
@@ -1,6 +1,23 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta http-equiv="Content-Security-Policy"  
+    content="default-src 'self';
+             connect-src 'self' https://*.tiles.mapbox.com https://api.mapbox.com https://osmstats-api.hotosm.org https://tasking-manager-tm4-production-api.hotosm.org https://matomo.hotosm.org https://hotosm.bamboohr.com;
+             font-src 'self' https://fonts.gstatic.com https://themes.googleusercontent.com;
+             frame-src https://embed.ted.com https://www.youtube.com https://docs.google.com;
+             frame-ancestors 'self';
+             img-src 'self' data: https://s3.amazonaws.com http://hotwww.s3-website-us-east-1.amazonaws.com https://api.monosnap.com https://cdn.hotosm.org https://monosnap.com https://resources.bamboohr.com https://s3.amazonaws.com https://www.colorhexa.com;
```

Comment:
Is `http://hotwww.s3-website-us-east-1.amazonaws.com` a correct url?

Reply:
it is a correct URL. It leads to the old HOTOSM website that ran on Drupal. There might be a legacy link to some old file or image.